api/src/models/team.model.js

```diff
@@ -63,6 +63,10 @@ const teamSchema = new mongoose.Schema(
       type: mongoose.SchemaTypes.ObjectId,
       ref: 'theme',
     },
+    teamImage: {
```

Comment:
Also, do we need to implement corresponding API functionality to make this PR complete?

Reply:
Was thinking of opening another PR for api functionality to keep this PR small.